src/plugins/percentage-fontsize.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * SCEditor Inline-Code Plugin for BBCode format
+ * http://www.sceditor.com/
+ *
+ * Copyright (C) 2011-2013, Sam Clarke (samclarke.com)
+ *
+ * SCEditor is licensed under the MIT license:
+ *	http://www.opensource.org/licenses/mit-license.php
+ *
+ * @fileoverview SCEditor percentage fontsize plugin for BBCode format
+ * This plugin allow usage of the percent indication in font size bbcode tag
+ * in a format used in phpBB system:
+ * [size=50]text[/font]
+ * means the text will be shown with font 50% of the current size
+ * [size=200]text[/font]
+ * means the text will be shown with font 200% of the current size
+ * @author Alex Betis
+ */
+
+(function (sceditor) {
+	'use strict';
+
+	var dom = sceditor.dom;
+
+	sceditor.plugins['percentage-fontsize'] = function () {
+		var base = this;
+
+		/**
+		 * Default sizes
+		 * @type {Object}
+		 * @private
+		 */
+		var sizes = [50, 75, 100, 150, 200];
```

Comment:
Does this behave like a constant? If so, turn the variable name into **CAPS**
Personally, I'd have a default value but allow setting options to change it

Reply:
good point. updating.

Reply:
Is there a documentation for plugin configuration fomat?
Couldn't find any documentation for format.js configuration for example...